test/test_ci_container.rb

```diff
@@ -27,23 +28,36 @@ def setup
       config.default_cassette_options = {
         match_requests_on:  [:method, :uri, :body]
       }
+      config.filter_sensitive_data('<%= Dir.pwd %>', :erb_pwd) { Dir.pwd }
     end
 
-    # Chdir to root, as Containment will set the working dir to PWD and this
-    # is slightly unwanted for tmpdir tests.
-    Dir.chdir('/')
-
     @job_name = self.class.to_s
     @image = 'ubuntu:15.04'
-    VCR.turned_off { cleanup_container }
```

Comment:
Probably because I was doing more stuff with VCR turned off, and then we changed things to not do multi line things anymore. So I've reverted the change